client/src/templates/Challenges/classic/desktop-layout.tsx

```diff
@@ -237,8 +239,8 @@ const DesktopLayout = (props: DesktopLayoutProps): JSX.Element => {
               orientation='horizontal'
             >
               <ReflexElement
-                flex={codePane.flex}
                 name='codePane'
+                {...(hasVerticalResizableCodePane && { flex: codePane.flex })}
```

Comment:
@ojeytonwilliams what do you think?
```suggestion
                {...{style: { flexGrow: codePane.flex, flexBasis: '100%'}}}
```

- - -

Btw. This doesn't work:
```js
              <ReflexElement
                name='codePane'
                flex={codePane.flex}
                {...{style: { flexBasis: '100%'}}}
                {...reflexProps}
                {...resizeProps}
              >
```

Reply:
I did a little digging into the Reflex source and it maps the `flex` property onto `flex-grow`. It doesn't really expect us to mess around with the details.

On balance, I think your original solution is the way to go. Sorry for the distraction!

Reply:
No worries!